src/content/1.7/basics/installation/nginx.md

```diff
@@ -157,7 +157,7 @@ server {
         deny all;
     }
     # Source code directories
-    location ~ ^/(app|bin|cache|classes|config|controllers|docs|localization|override|src|tests|tools|translations|travis-scripts|vendor|var)/ {
+    location ~ ^/(app|bin|cache|classes|config|controllers|docs|localization|override|src|tests|tests-legacy|tools|translations|travis-scripts|vendor|var)/ {
```

Comment:
I guess this line aims to protect installs that have been done using the source code instead of the ZIP build (because `tests` are removed in the ZIP build). So I complete it.

I thought about removing `travis-scripts` but it's better to keep it in case someone uses this configuration example for an old PrestaShop 1.7 version.